### PR TITLE
Voitecp task

### PR DESCRIPTION
**Pull request by VoitecP**
Partially finished task,

Created Order Model, storing present data from Cart model,  Simple model for Delivery method was created

### CODE REVIEW

File: Dshop/apps/core/api_views.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# from .models import Product
```
Comment (jacoor) [inline Dshop/apps/core/api_views.py:1]: What is this file? If not needed please remove.

File: Dshop/Dshop/urls.py [hunk 2]
```diff
@@ -40,7 +40,8 @@
     path('tinymce/', include('tinymce.urls')),
 
     path('api/users/', include('apps.users.api_urls')),
-    path('api/products/', include('apps.products_catalogue.api_urls')),
+    # path('api/products/', include('apps.products_catalogue.api_urls')),
```
Comment (jacoor) [inline Dshop/Dshop/urls.py:43]: Why this change? Everyhing was under /api/XXX now this change breaks it. Also changing url was not in scope of your tasks.

File: Dshop/apps/products_catalogue/migrations/0001_initial.py [hunk 3]
```diff
@@ -1,46 +1,122 @@
-# Generated by Django 4.2.3 on 2023-08-06 17:58
```
Comment (jacoor) [inline Dshop/apps/products_catalogue/migrations/0001_initial.py:1]: Please revert all the migration changes and add yours as new ones. Removing those migrations will break database for everybody in the team. 
Migrations are like version control (git) for DB changes. If you delete git commits, everyone else will have issues with the repo. Same thing here.

File: Dshop/apps/products_catalogue/fixtures/fixtures_products.json
Comment (jacoor) [inline Dshop/apps/products_catalogue/fixtures/fixtures_products.json:1]: What happened here? Why changes to fixtures?

File: Dshop/apps/products_catalogue/api_urls.py [hunk 4]
```diff
@@ -1,9 +1,11 @@
 from django.urls import path, include
-from .api_views import ProductViewSet
+from .api_views import ProductViewSet, OrderViewSet
 from rest_framework.routers import DefaultRouter
 
+
 router = DefaultRouter()
-router.register('', ProductViewSet, basename='products-api')
+router.register(r'products', ProductViewSet, basename='products-api')
```
Comment (jacoor) [inline Dshop/apps/products_catalogue/api_urls.py:7]: Why changing the url? The idea was that products list is on home page. That was not part of your task, please revert. 
Also, please review all the changes and stick to what was in the task: Order and Delivery method. Please revert everything else.

**Review by jacoor — CHANGES_REQUESTED** [review]
Lot's of weird changes. Please revert what was not in scope of your tasks.

**Comment by jacoor** [comment]
@VoitecP pease also fix conflicts. I believe that most of the conflicts might be gone once you revert the changes that should not be here.

**Comment by jacoor** [comment]
One more remark is for the descripion
- the title of the PR should mention what is the PR about
- the commits should be described by what the change is, not like "partially completed task". Please check here: https://github.com/pymasterspl/Dshop/pull/69 - every commit is described with what it does.

**Comment by VoitecP** [comment]
Pushed new branch, with correct name,   removed conflicts. This branch propably need's to be delete.

**Comment by jacoor** [comment]
deleted

**Pull request closed without merging**